recipes/thelink2012-any/all/conanfile.py

```diff
@@ -1,37 +1,54 @@
-from conans import ConanFile, tools
+# TODO: verify the Conan v2 migration
```

Comment:
What is this TODO in reference to? Leftover from when you were doing changes to the recipe? :)